chat-server/serverhtml.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
```

Comment:
It might be best to remove the html inside the name so its just server.html it helps make it more readable